docker.local/bin/build.miners.dev.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+GIT_COMMIT=$(git rev-list -1 HEAD)
+echo $GIT_COMMIT
+
+# Build it locally
+TOP="$PWD"
```

Comment:
Better this: `TOP=$(git rev-parse --show-toplevel)`, this way you can run your script from any place in the project

Reply:
Or `dirname $0` and then relative path from the folder with script, but git way is better IMHO.